src/processors/processVrmRequest.ts

```diff
@@ -0,0 +1,27 @@
+import { APIGatewayProxyEvent } from 'aws-lambda';
+import { getUserDetails } from '../services/user';
+import { TechrecordGet } from '../models/post';
+
+export const processPatchVrmRequest = (currentRecord: TechrecordGet, event: APIGatewayProxyEvent): Array<TechrecordGet> => {
+  const userDetails = getUserDetails(event.headers.Authorization!);
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+  const newVrm: string = JSON.parse(event.body!).newVrm as string;
```

Comment:
Also can we not just pass this in from the handler? Not sure it needs the whole event, so just pass in the parameter